crypto_secretbox.js

```diff
@@ -94,9 +93,8 @@ function crypto_secretbox_easy (o, msg, n, k) {
   const m = new Uint8Array(crypto_secretbox_ZEROBYTES + msg.byteLength)
   const c = new Uint8Array(m.byteLength)
   m.set(msg, crypto_secretbox_ZEROBYTES)
-  if (crypto_secretbox(c, m, n, k) === false) return false
+  crypto_secretbox(c, m, n, k)
   o.set(c.subarray(crypto_secretbox_BOXZEROBYTES))
-  return true
 }
```

Comment:
Shouldn’t this return a bool whether the decryption succeeded?

Reply:
this is the encryption method and it will always either throw or return true, since `crypto_secretbox` can only throw or return 0 - so either way the false check on L97 should change.  This return value is never checked elsewhere in the library or the tests either